UndertaleModTool/Controls/UndertaleObjectReference.xaml.cs

```diff
@@ -173,6 +215,33 @@ private void Details_Click(object sender, RoutedEventArgs e)
             }
         }
 
+        // TODO move this to the models
+        UndertaleCode CreationCode(UndertaleData data, string name)
```

Comment:
function name feels too generic.

This is adding a code entry, not creation code.
And I'm fairly certain we already have a function in the lib to do exactly that